setup.py

```diff
@@ -38,42 +38,42 @@
     },
     python_requires=">=3.10, <3.12",
     install_requires=[
-        "asgiref~=3.5.2",
-        "asteval==0.9.26",
+        "asgiref~=3.6.0",
+        "asteval==0.9.29",
         "async-timeout~=4.0.2",
-        "channels~=3.0.4",
-        "channels_redis~=3.4.1",
+        "channels~=4.0.0",
+        "channels_redis~=4.1.0",
         # Storage requirement for computing hashes.
         "crcmod",
-        "kubernetes~=21.7.0",
+        "kubernetes~=26.1.0",
         "docker~=6.1.1",
-        "Django~=3.2.12",
-        "djangorestframework~=3.13.1",
-        "django-filter~=21.1",
-        "django-versionfield~=1.0.2",
+        "Django~=4.2",
```

Comment:
Pin the version to `4.2.0`, otherwise `~=` operator will also install version 4.2 if it will ever exist.